patches/chrome-browser-ui-toolbar-media_router_contextual_menu.cc.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/chrome/browser/ui/toolbar/media_router_contextual_menu.cc b/chrome/browser/ui/toolbar/media_router_contextual_menu.cc
+index 46b5dd8c2b246bb830ce3ae38e977d94d051d64e..29c300491aeb6c5ff4f5adb181bd77954e6339c4 100644
+--- a/chrome/browser/ui/toolbar/media_router_contextual_menu.cc
++++ b/chrome/browser/ui/toolbar/media_router_contextual_menu.cc
+@@ -135,12 +135,18 @@ bool MediaRouterContextualMenu::IsCommandIdVisible(int command_id) const {
+ 
+ void MediaRouterContextualMenu::ExecuteCommand(int command_id,
+                                                int event_flags) {
++#if defined(BRAVE_CHROMIUM_BUILD)
```

Comment:
maybe it would be better to handle these by changing the url in a ShowSingletonTab override? Subclassing this will still require patches because they are created by static methods, but ShowSingletonTab is easily overriden using chromium_src preprocessor method

Reply:
yes, good idea.